mage_integrations/requirements.txt

```diff
@@ -4,6 +4,7 @@ backoff==2.0.0
 clickhouse_sqlalchemy
 couchbase==4.1.1
 deltalake==0.7.0
+duckdb==0.8.1
```

Comment:
it should be added to `requirements.txt` instead of `mage_integrations/requirements.txt`
you should also add it to setup.py